app/assets/javascripts/timeline.js

```diff
@@ -142,9 +142,9 @@ samson.filter("localize",
       var localDate = new Date(parseInt(ms));
 
       var day    = DAYS[localDate.getDay()],
-          year   = localDate.getFullYear(),
-          date   = localDate.getDate(),
-          month  = MONTHS[localDate.getMonth()];
+        year   = localDate.getFullYear(),
```

Comment:
I've never seen the dont-indent style, but I'll change the indent back. My
IDE decided to do this anyway! :)
On 6 May 2015 1:21 am, "Roman Sandler" <notifications@github.com> wrote:

> In app/assets/javascripts/timeline.js
> <https://github.com/zendesk/samson/pull/394#discussion_r29726847>:
>
> > @@ -142,9 +142,9 @@ samson.filter("localize",
> >        var localDate = new Date(parseInt(ms));
> >
> >        var day    = DAYS[localDate.getDay()],
> > -          year   = localDate.getFullYear(),
> > -          date   = localDate.getDate(),
> > -          month  = MONTHS[localDate.getMonth()];
> > +        year   = localDate.getFullYear(),
>
> IMO either don't indent or align with the first variable name
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/394/files#r29726847>.
>